client/packages/portal-ui/src/portal-menu/styles.ts

```diff
@@ -10,12 +10,13 @@ export const styles = {
 		height: 24px;
 	`,
 	groupWrapper: css`
+		align-items: flex-start;
+		break-inside: avoid;
 		display: flex;
-		-webkit-flex-direction: column;
```

Comment:
This fixes the issue yes but causes a bigger issue. select the OGP context and open the menu. It is now broken. We need to find a better solution. 

Therea are calculation on break points in the menu i think we need to utilize this at some point. 
But if you can fins a good solution to fix both the menu and the app list here then be my guest ;)

Have a look at the PortalMenu.tsx file in the portal-client project.